_summaries/ulyanov2016instance_2.md

```diff
@@ -1,19 +1,49 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: ulyanov2016instance
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: x / 10
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+  
+Ulyanov et. al. revist the task of image stylization. Previous works like Gatys et. al. (2016) have introduced methods for style transfer with good results but high computational cost.  Ulyanov et. al. (2016) introduced equivalent feed-forward generator networks, however these networks did not achieve results as good as those propsed in Gatys et. al. In this paper Ulyanov improves his generator architecture through the replacement of batch normalization layers with instance normalization layers and keep them at test time. 
```

Comment:
I think the summary is good but i don't think it is not necessary to include the author's names. You may as well go straight to what the problem is and how it was solved.